docs/lib/datastore/fragments/js/sync/50-selectiveSync.md

```diff
@@ -134,4 +160,12 @@ DataStore.configure({
     })
   ]
 });
-```
\ No newline at end of file
+```
+
+In order to construct a query expression, you will need to return a predicate with the primary key of the GSI. You can only use the `eq` operator with this predicate.
```

Comment:
To construct a query expression, return a predicate with the primary key of the GSI.